packages/discord.js/src/managers/StageInstanceManager.js

```diff
@@ -59,14 +61,17 @@ class StageInstanceManager extends CachedManager {
     const channelId = this.guild.channels.resolveId(channel);
     if (!channelId) throw new DiscordjsError(ErrorCodes.StageChannelResolve);
     if (typeof options !== 'object') throw new DiscordjsTypeError(ErrorCodes.InvalidType, 'options', 'object', true);
-    let { topic, privacyLevel, sendStartNotification } = options;
+    let { guildScheduledEvent, topic, privacyLevel, sendStartNotification } = options;
```

Comment:
I'm confused as to why this is a `let`, it doesn't seem to be reassigned anywhere.